valkey.conf

```diff
@@ -2602,8 +2602,11 @@ jemalloc-bg-thread yes
 # Set aof rewrite child process to cpu affinity 8,9,10,11:
 # aof-rewrite-cpulist 8-11
 #
-# Set bgsave child process to cpu affinity 1,10,11
+# Set bgsave child process to cpu affinity 1,10,11:
 # bgsave-cpulist 1,10-11
+#
+# Set slot migration child process to cpu affinity 1,8,9,10:
+# slot-migration-cpulist 1,8-10
```

Comment:
Since slot migration and full-sync bgsave are functionally the same operation, I would have expected slot-migration-cpu list to use the same cpu-list as bgsave. I would rather not add another configuration for this.

To get this PR merged, I would like to suggest we don't add the new config right now.